src/main/java/hudson/plugins/sloccount/util/FileFinder.java

```diff
@@ -9,6 +9,7 @@
 import org.apache.tools.ant.BuildException;
 import org.apache.tools.ant.Project;
 import org.apache.tools.ant.types.FileSet;
+import org.jenkinsci.remoting.RoleChecker;
```

Comment:
I think that RoleChecked should stay (it is type of checkRoles argument).
However it looks that FilePath.FileCallable (imported few lines above) shall be replaced by  MasterToSlaveFileCallable , shan't it?